openedx_webhooks/lib/github/utils.py

```diff
@@ -42,7 +41,7 @@ def repo_name(repo):
     return "{0.owner.login}/{0.name}".format(repo)
 
 
-def create_hook(repo, config, events, active, dry_run=False):
+def create_hook(repo, config, events, active, dry_run=False):  # pragma: no cover
```

Comment:
You've got 7 or 8 lines of code here, including two conditionals, string formatting, and exception raising. You would test that all that works. You have tests of GithubWebHookEvent, and all of its methods are one or two lines with no conditionals. I don't understand why that code is interesting to test, and this code is not?